style.css

```diff
@@ -1,595 +1,562 @@
 @media only screen and (min-width: 300px) {
```

Comment:
- [OPTIONAL] As this is a mobile-first approach you don't need to do a media query for mobile. You can later add a media query for the desktop version when it is required 😄

Reply:
Thanks for the reply, Can I use awesome font for my WIFI also

Reply:
@GiuseppeTG Thanks for the reply, Can I use awesome font for my WIFI also

Reply:
@GiuseppeTG can I also use font for my Signal, and battery